core/domain/user_services.py

```diff
@@ -185,9 +329,16 @@ def get_user_settings_from_username(username):
 def get_users_settings(user_ids):
     """Gets domain objects representing the settings for the given user_ids.
 
-    If the given user_id does not exist, the corresponding entry in the
-    returned list is None.
+    Args:
+        user_ids: list(str). The list of user_ids to get UserSettings
+            domain objects for.
+
+    Returns:
+        list(UserSettings). The UserSettings domain objects corresponding to
+            the given user ids. If the given user_id does not exist, the
```

Comment:
Deindent this and the next line by 4, for consistency with the other "Returns" docstrings.